tests/Shipment/ShipmentTest.php

```diff
@@ -222,6 +222,15 @@ public function test_it_should_transform_shipment_to_json_api_with_all_inputs():
             ],
         ]);
 
+        $taxNumberMock = Mockery::mock(TaxIdentificationNumbers::class, [
+            'toArray' => [
+                'country_code' => 'UK',
+                'Description'  => 'The IOSS number for the Kazan company',
+                'number'       => 'KSI818298',
+                'type'         => 'ioss',
+            ],
```

Comment:
I think that this is actually wrong. It represents a single item while it should be an array of items.
```suggestion
            'toArray' => [
                [
                    'country_code' => 'UK',
                    'Description'  => 'The IOSS number for the Kazan company',
                    'number'       => 'KSI818298',
                    'type'         => 'ioss',
                ]
            ],
```

Reply:
You are absolutely right, this is sloppy of me, I knew this.